### PR TITLE
doc(examples): add sweep examples

### PR DESCRIPTION
**Pull request by vxio**
Added examples for:
1. sweep config crud endpoints
2. sweeps list + get

### CODE REVIEW

File: examples/sweeps/sweep_test.go [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+		fmt.Printf("new client: %v", err)
+		return
+	}
+
+	var (
+		accountID = "2f29c185-91d8-40c7-8963-3915aa673c21"
+		walletID  = "ce906d91-61ef-47eb-8e15-38f80b863200"
+
+		ctx        = context.Background()
+		minBalance = "100.0"
```
Comment (adamdecaf) [inline examples/sweeps/sweep_test.go:22]: I know one decimal is supported, but should we show `100.00` just to suggest the more natural format?

File: examples/sweeps/sweep_test.go [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+	"context"
+	"fmt"
+
+	"github.com/moovfinancial/moov-go/pkg/moov"
+)
+
+func Example_SweepConfigs_Endpoints() {
+	mc, err := moov.NewClient()
+	if err != nil {
+		fmt.Printf("new client: %v", err)
```
Comment (adamdecaf) [inline examples/sweeps/sweep_test.go:13]: t.Error / t.Fatal ? We can't proceed without a client.

File: examples/sweeps/sweep_test.go [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+		AccountID:           accountID,
+		SweepConfigID:       sweepConfig.SweepConfigID,
+		MinimumBalance:      &minBalance,
+		PullPaymentMethodID: &pullPaymentMethodID,
+	})
+	if err != nil {
+		fmt.Printf("udpating sweep config: %v", err)
+		return
+	}
+	fmt.Printf("updated sweep config: %+v", sweepConfig)
```
Comment (adamdecaf) [inline examples/sweeps/sweep_test.go:61]: t.Logf ?

Comment (vxio) [reply examples/sweeps/sweep_test.go:61]: similar comment [above](https://github.com/moovfinancial/moov-go/pull/144#discussion_r1757187575)

File: examples/sweeps/sweep_test.go (hunk 1 above)
Comment (vxio) [reply examples/sweeps/sweep_test.go:22]: Yeah, that makes sense, updating

**Review by sarumont — APPROVED** [review]

**Review by carsonab — APPROVED** [review]

File: examples/sweeps/sweep_test.go (hunk 2 above)
Comment (adamdecaf) [reply examples/sweeps/sweep_test.go:13]: Go's Example funcs can be used as tests, by [specifying output](https://github.com/moov-io/ach/blob/master/examples/example_ackRead_test.go#L54) that's expected. 

We've used Tests elsewhere in examples to leverage `*testing.T`.

Comment (vxio) [reply examples/sweeps/sweep_test.go:13]: Yeah, but in this example, all of the IDs are just randomly generated UUIDs so this func is never meant to be run -- moreso an example users can use as a reference as they're building their integration.

**Review by adamdecaf — APPROVED** [review]

**Pull request merged**